geometry_msgs/Point agent_pose  # linear x, y, and angle z
geometry_msgs/Point agent_twist # linear x, and angle z
geometry_msgs/Point goal_pose # x, y only
sensor_msgs/LaserScan scan
float32 reward
